lib/ffmpeg/encoderbase.py

```diff
@@ -57,6 +57,7 @@ def hwformat(self):
   level         = property(lambda s: s.ifprop("level", " -level {level}"))
   ladepth       = property(lambda s: s.ifprop("ladepth", " -look_ahead 1 -look_ahead_depth {ladepth}"))
   forced_idr    = property(lambda s: s.ifprop("vforced_idr", " -forced_idr 1 -force_key_frames expr:1"))
+  tcbrc_idr     = property(lambda s: s.ifprop("vtcbrc_idr", " -low_delay_brc 1 -low_power 1"))
```

Comment:
tcbrc is a BRC mode, its name should not include idr which will mistake others

Reply:
Ok, I have revised and resubmitted